Va gov/feature/13411: Updates to Health Care Left Nav

accbjt opened the pull request:
## Description
Updates to Health Care Left Nav

## Testing done
Manual

## Acceptance criteria
- [ ]  Updates needed to left nav based on completion of all hubs and URLs are documented in the LeftNav Links document and highlighted in bright green.

https://docs.google.com/spreadsheets/d/1SsA5RVkERpMF8DE4Oal7fy4dK617JkdJgMQ2zrRdJXc/edit?usp=sharing

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

cvalarida committed:
Revert "Deleted the email-gathering page and its javascript (#8153)" (#8239)

This reverts commit 383a5402b43029144b84d4c9e22d6d059d182bc0.

cvalarida committed:
Check for .then() instead of instanceof Promise (#8197)

* Check for .then() instead of instanceof Promise

* Check for truthiness

Commit:
Evss issue 12574 (#8241)

* Replace successfulSync with syncStatus

* Update tests

* Add in sync tests

* Fix broken tests

markgreenburg committed:
526 remove old requireds mg (#8215)

Removes some dead code. Someone else already bumped package.json so we didn't need to do that in this PR to get the latest schema.

cvalarida committed:
Consolidate pages into index.js (#8221)

* Consolidate pages into index.js

* Named export

* Don't use ../

* Move the new pages

Commit:
Rename complaint tool as feedback tool (#8248)

* Update vets-json-schema version

* Update formId

* Update vets-json-schema version

* Update helpers, files, folders, components and other instances of complaint tool

* Update docs

* Remove obsolete ignore

* Use COMPLAINT-TOOL as formId until BE updates as well

* Nest facility code in view:searchSchoolSelect

bshyong committed:
Ihub widget (#8243)

* ihub appointments widget

* update unit test

* only gate in production

jstanek committed:
Update Fry Scholarship time limits (#8230)

* Update Fry Scholarship time limits

* Spelling correction

* Copy edits

* ME review: EDU FRY

markgreenburg committed:
All claims 1dot4 mg (#8238)

Adds chapter 1, section 4 to the All Claims Form. This section contains questions on separation / severance pay as well as training pay.

ncksllvn committed:
Add flag for preventing API request in prod (#8268)

jbalboni committed:
All claims 1.5 service info (#8229)

erikphansen committed:
Send the same school data regardless of school input method (#8267)

* Send the same school data regardless of school input method

* Convert "USA" to "United States"

Commit:
Replace complaint with feedback (#8272)

jstanek committed:
Adding asterisks back to clarify eligibility (#8240)

* Adding asterisks back to clarify eligibility

* ME review: adaptive housing grants

* Update adaptive-housing-grants.md

* Update adaptive-housing-grants.md

* Update adaptive-housing-grants.md

ncksllvn committed:
Implement feature flag and build command for brand-consolidation (#8255)

* Add flag IS_BRAND_CONSOLIDATED_BUILD into Metalsmith metadata and Webpack

* Add a platform module for feature flag in Webpack

* Change module to a function instead

* Add unit test

* Change naming conventions

* Move feature flag to top-level directory

* add the components directory

* Keep Formation at master version

* Update watch command for devpreview

* Change unit test file extension

* Force feature-flag parse to boolean

* Revert "Force feature-flag parse to boolean"

This reverts commit 90650cd0cc556f8da019c6069fc8ff0b06022e1b.

* Switch options to env variable where applicable

* Switch Webpack to env variable for brand-consolidation flag

* Add a comment explaning why we can't rely on booleans here

* fix inconsistency

* Rename build settings and opt for that instead

* Remove flag from metalsmith

* Move flag to parent property window.settings

markgreenburg committed:
526 sip date format mg (#8269)

Update the date formatting for Save In Progress intros and footers to match rest of vets.gov.

erikphansen committed:
Add prefill messaging to GIBFT and track changes to On Behalf Of... (#8225)

* Add prefill message to GIBFT

* Track changes to 'on behalf of...'

Created new UserInteractionRecorder component to track selected user
actions as they occur.

erikphansen committed:
Update feedback form config for new JSON schema (#8277)

markgreenburg committed:
526 remove special issues mg (#8275)

Removes specialIssues from the 526 form, since they're not supported by the submit endpoint for mvp.

cvalarida committed:
526 confirmation poll (#8113)

* First draft

* Fetch mocked

* Allow for chaining of `apiRequest`

* Short circuit polling

* Add tests

* Use the real statuses and endpoint

* Return a thing.

* Fix payment info bug

* WiP

* Messaging

* Confirmation page and polling component swapped

* Use the jobId

* PR comments

* Move content

* Tests

* Uncomment the disabilities list

ncksllvn committed:
Build Settings - Integrate build-created settings into Redux; add brand-consolidation containers (#8274)

* Add flag IS_BRAND_CONSOLIDATED_BUILD into Metalsmith metadata and Webpack

* Add a platform module for feature flag in Webpack

* Change module to a function instead

* Add unit test

* Change naming conventions

* Move feature flag to top-level directory

* add the components directory

* Keep Formation at master version

* Update watch command for devpreview

* Change unit test file extension

* Force feature-flag parse to boolean

* Revert "Force feature-flag parse to boolean"

This reverts commit 90650cd0cc556f8da019c6069fc8ff0b06022e1b.

* Switch options to env variable where applicable

* Switch Webpack to env variable for brand-consolidation flag

* Add a comment explaning why we can't rely on booleans here

* fix inconsistency

* Rename build settings and opt for that instead

* Remove flag from metalsmith

* Add the reducer

* Move flag to parent property window.settings

* Add a container for brand-consolidation components

* Add a Vets.gov container

* Move flag to parent property window.settings

* Add unit test

* Add Vets.gov test

* Update tests to use .render for consistency

* Add additional assertion for mapStateToProps; change prop name to vetsDotGovEnabled

* Update JSDoc

cvalarida committed:
Revert "526 confirmation poll (#8113)" (#8283)

This reverts commit 6426274096afee318b76f3673f7e891f2cc98bc1.

erikphansen committed:
Fix aXe test failure on SchoolSelectField (#8281)

* Fix aXe test failure on SchoolSelectField

* Re-disable the tests

cvalarida committed:
526 confirmation poll take 2 (#8284)

* First draft

* Fetch mocked

* Allow for chaining of `apiRequest`

* Short circuit polling

* Add tests

* Use the real statuses and endpoint

* Return a thing.

* Fix payment info bug

* WiP

* Messaging

* Confirmation page and polling component swapped

* Use the jobId

* PR comments

* Move content

* Tests

* Uncomment the disabilities list

* Fix tests

* Add tests

* Revert "Revert "526 confirmation poll (#8113)" (#8283)"

This reverts commit 8883a43e32bcee38b1139d55d6841cbe0403a62c.

* Fix tests

U-DON committed:
Fixed redirect issues with MHV upgrade flow. (#8224)

ncksllvn committed:
Add Metalsmith middleware for processing "aliases" property into redirects (#8293)

* Add redirects middleware

* Switch to path property instead of generating

* Clean up code

* Fix for redirect to index

U-DON committed:
Fixed incorrect object path resulting in incorrect last updated date for facility services. (#8292)

jstanek committed:
Content/wip/aug avg disability claim days (#8297)

* Average days update

* Updating average number of days and month

erikphansen committed:
Upgrade design-system/formation (#8298)

* Upgrade design-system/formation

* Upgrade design-system/formation to 1.4.8

cvalarida committed:
Open redux dev tools on staging (#8299)

markgreenburg committed:
All claims 2dot1 mg (#8285)

Adds the Rated Disabilities picker to the 526 All Claims form

Commit:
Upgrade formation (#8307)

markgreenburg committed:
526 va facility required mg (#8305)

Update vaFacility schema for 526 Increase form so that users can't skip the page anymore.

markgreenburg committed:
add better validation messages to address page (#8310)

Validation messages for 526 Increase address errors are now human-readable

ncksllvn committed:
Reconcile scripts directory with brand-consolidation (#8302)

* Reconcile script with build-consolidation

* Remove unneeded check

* Add va-gov directory

* Remove files from ignore list

* Add old healthcare url to watch list

ncksllvn committed:
Add foresee script for staging (#8313)

U-DON committed:
Brand consolidation feature flag for relevant user nav components (#8316)

* Brand consolidation feature flag for relevant user nav components.

* Add safety check for window.settings

Commit:
update copy (#8330)

* update copy

* remove space; open link in new tab

cvalarida committed:
Add aria-label (#8332)

Commit:
Feedback tool school search axe fixes (#8319)

* Move search field to feedback folder

* manage focus on school search pagination

* Support optional aria label specifier for improved accessibility

* Update school select field unit test to check that form data is reset properly

* PR comment: use ariaLabelSuffix to specify how this prop is used

* Update formation version

* Reset school search fields to null, to ensure no values are stored in the form data

jbalboni committed:
All claims 2.2 (#8290)

accbjt committed:
add medallia script to the footer includes [WIP DO NOT MERGE] (#8260)

* add medallia script to the footer includes

* fix for button changing locations on focus

* add css for fixing focus on buttons

* fix for div button medallia width of 81

* Disable in production

erikphansen committed:
Add label above the manual school input checkbox (#8340)

* Add label above the manual school input checkbox

* Curly apostrophe

* Restore labelAboveCheckbox

jbalboni committed:
Update error message on 526 form (#8346)

markgreenburg committed:
526 ITF message updates (#8329)

Updates ITF status message contents

jbalboni committed:
Create beta enrollment process for 526 (#8266)

* WIP without content

* Update names

* Minor tweaks

* Tweaks, pr comments

* Better workflow

* Remove semicolon

* Hide while loading

* Fix axe issue

* PR comments

* Add test

* Put beta gate around intro page buttons

Commit:
add online learning and distance learning filters (#8376)

markgreenburg committed:
526 add no evidence mg (#8375)

Adds 'no evidence to submit' option to 526 Increase form

ncksllvn committed:
Medallia/Foresee - Add Metalsmith logic for determining JavaScript snippets for survey tools (#8382)

* Add foresee production script

* Add script for processing which survey script to embed

* Limit to Staging

* Add the "split" filter

* Filter entryname list

* Change variable name based on new reqs

* Switch to ==

cvalarida committed:
Simplified the PR template (#8301)

* Simplified the PR template

* Removed alerts line

Commit:
Feedback tool profile card (#8380)

* Specify feedback form dashboard form ID

* Add year and remove leading zeros

* Add periods to meridiem

* Update test

Commit:
update content (#8374)

jstanek committed:
Fixing typo in MVI alert box message (#8336)

* Fixing typo in MVI alert box message

* Removing "with" typo in MVI warning message

* Removing "with" typo in warning message

* Removing period in alert message title

erikphansen committed:
Change handling of school addresses from the search tool (#8327)

* Change handling of school addresses from the search tool

- Combine the second and third address fields so no data is lost; this
is especially important for some international addresses that the search
tool returns as the many of those contain important address info in the
`address3` field.
- If the address from the search tool is domestic, transform the country
to the prop enum value, "United States".
- If the address is not domestic, send the country as-is and add the
`viaSearchTool=true` field on the `address` so the backend will use the
school address schema that accepts any string for the country value.

* Use the `street3` field

* Refactor address transforms to a helper

* Bypass the JSON stringify transformer

The default transformer was completely removing address objects because
it was very aggressively removing address objects that it deemed
invalid. In this case that meant if the address did not contain a zip
code/postal code, the entire address would be removed from the school
object

* prevent sending search tool address fields that are too long

* Remove `viaSearchTool`, upgrade vets-json-schema

* refactor string trimming

Commit:
Pull latest from bc branch

Commit:
Add updated Jenkinsfile

Commit:
Revert "Add updated Jenkinsfile"

This reverts commit 6c665af1cb91bff09bdf9042ddc46605d779d596.

Commit:
Update Jenkinsfile devpreview to vagovdev

markgreenburg committed:
All claims 41 mg (#8388)

Adds address page, payment info page to 526 All Claims. Move most related files to All Claims and point the Increase Only pages at them.

Commit:
Remove vagov entirely

ncksllvn committed:
Merge pull request #8393 from department-of-veterans-affairs/master-reconcile-brand-consolidation-scripts

Pull latest script config from brand-consolidation branch

Commit:
add tabbable title (#8379)

* add tabbable title

* remove class and id cruft

* update name of helper function

Commit:
Remove temporary prefill available logic (#8389)

* Remove temporary clause for prefill check

* Remove obsolete prefillAvailable configuration

accbjt committed:
Add Hurricane Florence warning banner (#8398)

* add Typhoon alert

* update copy for hurricane alert

* add https to link tag

ncksllvn committed:
Update Yarn's Formation (#8414)

* Update Yarn's Formation

* Bump to 1.4.12

cvalarida committed:
ReviewCardField -- Render data type field on review (#8403)

* Render data type field on review

* Unit test

* Use onReviewPage instead of reviewMode

* Set onReviewPage in selector

Commit:
Provide issue examples (#8377)

* Provide issue examples.

* change markup

* Update page description to mention examples

* Specify description context

jbalboni committed:
Add titles for 526 evidence pages on review (#8404)

jbalboni committed:
526 show summary of evidence on review page (#8405)

jkassemi committed:
Additional image optimization, lower resolution (#8412)

markgreenburg committed:
526 fdc content mg (#8392)

Updates Fully-Developed Claim Program helper text on the 526 Increase form

markgreenburg committed:
Add loading state svg to vets-website (#8395)

Fixes the missing spinner asset on vets.gov. Incidentally, also updates the spinner vets.gov uses from the old yellow circle with vets.gov gif to the new blue spinner circle svg.

jkassemi committed:
veterans.jpg image optimization (#8406)

Provides several optimized variations, and opts to replace the
production version with a guetzli Q84 optimized variant for a homepage
bandwidth savings of ~116K.

```
117K veterans-guetzli-84.jpg
158K veterans-guetzli-88.jpg
180K veterans-guetzli-92.jpg
206K veterans-guetzli-96.jpg
191K veterans-jpegmini.jpg
233K veterans-original.jpg
```

accbjt committed:
Update Hurricane Florence homepage banner (#8421)

Commit:
Feedback error messages (#8416)

* Make error messages more specific

* Update vets-json-schema version

* Sort lines

erikphansen committed:
Only show prefill message on pages that were prefilled (#8435)

* Only show prefill message on pages that were prefilled

* Work in PR feedback

jbalboni committed:
Remove lodash/fp from platform and static pages code (#8436)

cvalarida committed:
Use the review PhoneNumberWidget in ui:reviewWidget (#8422)

* Use the review TextWidget in ui:reviewWidget

* Use PhoneNumberReviewWidget

Commit:
update link (#8443)

ncksllvn committed:
Remove reference to Pacific Islands (#8450)

ncksllvn committed:
Isolate Metalsmith build from Webpack build (#8396)

* Add environments constant

* Move environments to constants directory

* Add webpack-connect-helpers

* Move environment import

* Add process variables to build settings

* Create middleware for filtering by environment

* remove unused metalsmith-ignore

* Isolate webpack logic to the config directory

* Fix build problems

* Remove extra code

* Restore create-build-settings

* Move manifest-helpers to parent

* Fix up link checker

* Formatting

* Fix brackets

* Fix signature of link check middleware

* Add aliases attribute to education/apply now that metalsmith-redirect is removed

* Add "apply for edu benefits" to broken-link exceptions

* Remove other unused metalsmith modules and regenerate yarn

* Rename process-file-manifest

* Update docs

markgreenburg committed:
All claims 42 mg (#8438)

Homelessness / at-risk form page for 526 All Claims form

Commit:
Provide currency format validation (#8439)

* Provide currency format validation

* Add validateCurrency tests

Commit:
Feedback school search/submission analytics (#8442)

* Add can’t find school analytics event

* Add submission failure analytics event

* Rename feedback tool analytics code

* Fix trackingPrefix

* Use tracking prefix to compute strings

* Remove obsolete dash

erikphansen committed:
Use `apiRequest` helper in GIBFT `submit` (#8448)

* Use `apiRequest` helper in GIBFT `submit`

* remove default value for `optionalSettings`

* Undo last commit

* More robust mocking of the repsonse.headers

ncksllvn committed:
Change EVSS to vet360 (#8194)

erikphansen committed:
Update checkStatus to use apiRequest helper (#8464)

erikphansen committed:
Use apiRequest helper in burials helpers (#8465)

erikphansen committed:
Use apiRequest helper in vic2 helpers (#8466)

erikphansen committed:
Add `Last 4 of SSN` field to GIBFT (#8479)

jbalboni committed:
Minimize css in prod environments (#8478)

Commit:
Fix trackingPrefix import (#8471)

* Fix trackingPrefix import

Commit:
update ui field and classnames (#8485)

* update ui field and classnames

* adjust unit test to not check for legend

ncksllvn committed:
Provide file metadata (front matter) to app manifests to enable dynamic root URLs (#8470)

* Add environments constant

* Move environments to constants directory

* Add webpack-connect-helpers

* Move environment import

* Add process variables to build settings

* Create middleware for filtering by environment

* remove unused metalsmith-ignore

* Isolate webpack logic to the config directory

* Fix build problems

* Remove extra code

* Restore create-build-settings

* Move manifest-helpers to parent

* Fix up link checker

* Formatting

* Fix brackets

* Fix signature of link check middleware

* Add aliases attribute to education/apply now that metalsmith-redirect is removed

* Add "apply for edu benefits" to broken-link exceptions

* Remove other unused metalsmith modules and regenerate yarn

* Rename process-file-manifest

* Update docs

* Convert build settings to standard middlware

* Move build-settings middleware to bottom of build.js

* Apply all frontmatter into app settings

* Allow the JS extension in manifests

* Add a custom loader for manifests

* Change function name to receiveContentProps

* Update Messaging manifest

* Remove unneeded prop from messaging

* Update Burial manifest

* Update Discharge Wizard

* Update GI manifest

* Update HCA manifest

* Update Letters manifest

* Update post-911 gibs

* Update RX manifest

* Update VIC V2

* Update 686

* Update all-claims

* Update edu benefits feedback tool manifest

* Update veteran representative

* Update chapter 31 manifest

* Update chapter 36 manifest

* Fix conflicts with manifest.jsons

* Move redirects to after sitemap

* Rewrite the rewrite functions

* Rework middleware to execute with build-options inside callback

* remove unneeded manifest checks

* fix import

* Fix lint errors

* remove get-routes

* Refactor parser for app settings

* Add e2eTestDisabled property

* Update data structure to use "contentProps"

* Spruce up the loader

* Bind settings to options

accbjt committed:
update health-care left nav

accbjt committed:
update health-care and track-claims to claim-or-appeal-status

accbjt committed:
revert back to track-claims

accbjt commented:
@mnorthuis We have a conflict with one of the paths. The recent path change to put things back to the vets-gov path is having conflicts with` track-claims`. On the URL map spreadsheet it says to revert back to `track-claims` but in the Left Rails Nav has `claim-or-appeal-status`. In this PR I changed it to `track-claims`. Please clarify what this should be before we merge the PR.

cc: @ncksllvn @RobAWilkinson

accbjt committed:
add / to track-claims

ncksllvn committed:
Add brand-consolidation readme (#8488)

* add readme

* Add zenhub/sprint info

* Add note about feature flag

* Add — to build command

* Fix typo

ncksllvn committed:
Merge brand-consolidation assets directory (#8512)

* Add brand-consolidation assets

* Remove .ai files

ncksllvn committed:
Add brand-consolidation Metalsmith directory (#8513)

* Add brand-consolidation Metalsmith directory

* Remove index files conflicting w/other paths

* Fix broken links after running modifed link checker

mnorthuis commented:
The left rail spreadsheet is correct.  There is a static page for /claim-or-appeal-status/ that the navigation points to, this is accurate.  The actual tool would have been /claim-or-appeal-status/status/.  That is the URL that should have been reverted.  

In all hubs, if there is a link to Check Claim or Appeal Status, it will link to the new static page at /claim-or-appeal-status/.  That static page will then link to the tool at the reverted URL /track-claims/

ncksllvn committed:
Integrate brand-consolidation with master (#8511)

* fixes vertical centering of menu button on mobile

* commented out sass

* adjusted health benefit to use new left rail nav using metalsmith

* modify trigger to use non react component and added a div to contain left rail nav on mobile

* made icon for drop down smaller

* updating content

* bolding

* tweaking copy

* Add success_relay to sessions API hit (#8333)

* Move the files to their new locations

* ME review and copyedit: check-post-9-11-gi-bill

* want to use success_relay=preview_vagov for preview & dev-preview (#8343)

* Tweak to check-post-9-11

*  Fixed invisible sign-in modal close button (#8345)

* Fixed invisible sign-in modal close button.

* More general rule for all modal close buttons.

* add new lower environment names to build configs (#8322)

* incorporating updates from wip vets.gov page

* adding collection

* adding </div>

* tweaks to new text added: check-gi-bill

* Run script

* Fix links to missing pages

* creating placeholder blank page

so url is accounted for

* ME review and copyedit: how to use benefits

* fixed testing fees link label and url: how to use benefits

* Update the collection

* adding file (#8276)

* adding file

* revising call to action module

* tweaks

* revising blue module copy

* minor tweak

* minor tweak

* ME and copyedit complete: upload supporting evidence

* tweak: uploading supporting evidence

* bugfixes for footer

* Take the test-fees from /education/advanced-training-and-certifications/licensing-certification/

* add healt-care external pages

* Add stub for gi-bill

* fix affordable display name

* disability hub links

* Eligibility section of Disability hub working

* Populate edu links; add stubs for missing pages

* manage benefits disability sidenav

* side nav for disability complete

* targets blank for disability hub

* Blank out test fees (#8351)

* bold sign in link

* fix page links for health-care for left rail nav

* added get benefit section of burials

* add plan a burial and more resources sections for burials

* removed target from pages that a va sub domains

* Overwrite stub

* Add the aliases property

* Remove Apply Now link and description (#8320)

* fixing and renaming link (#8309)

* Removed "Apply Now for VR&E" link and description (#8323)

* Move /burials to /burials-memorials and update links

* FIx broken links

* Splitting link to survivor and dependents into 2

to align with new IA

* adding <br>

* rename and refactor how-to-file-a-claim into how-to-file-claim

* set up next and previous to fire to outside pages

* fixed broken links

* add handshake icon

* disability font-awesome icon

* bug in navigation prev-next

* font-size and padding tweak

* activate links for burial, eduction, and disability in the megaMenu

* Remove extra divs

* checks titles and orders

* pow title

* fixes bad links on diability hub page

* PR comments

* revert changes to disability-benefits page

* Activate crisis-line modal

* Remove double USWDS import

* Update references to Military Funeral page

* Delete burials-and-memorials directory

* Update references to /dependency-indemnity-compensation/

* gets More Resources side-nav

* Manage Benefits sidenav complete

* font icon

* Delete disability-benefits directory

* Update refs to links missing from mappings spreadsheet

* Delete dir again after pulling latest

* Remove outdated collection

* Update Burials landing

* Update refs to View VA Payment History

* Add link values for Disability and Edu sections

* Consolidate education-training with education

* Remove unneeded props

* add education links in left nav rail

* fix for left side nav logo and title centering

* Remove links according to spreadsheet

* upgrade formation

* remove errant major links: housing

* Update index.md

* Fix for React-education routes

* Add how-to-apply to edu pages

* Add stub for GI Bill

* Remove italics from "e" in My HealtheVet (#8354)

* Brand-Consolidation: Disability titles and links to records and burials (#8362)

* Update titles and links

* hides /disability/add-remove-dependent/add-dependent-form-21-686c/ from the sidebar

* /records pages become #

* Add stub for gi-bill

* content/wip/Va gov content wip burials hub sme changes (#8367)

* incorporating sme changes

Requested content changes to Burials hub page, from NCA on Aug. 7:

- Remove the word "national" promo spot teaser text on hub page
- Remove the word "your" in last sentence of hub page intro text: "...and how to plan for a burial in advance or at your time of need."
- Remove the word "your" in the Pre-Need Burial teaser text: "...and make the process easier for your family members in their time of need."

* incorporating sme change

- remove "national" from promo

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* lowercase "with" in "Connect with Us"

* Delete /education/gi-bill/

* remove crossed out links from left rail nav (#8371)

* Brand Consolidation: Blind redirects for health tools (#8373)

* Corrected typo.

* Initial setup for CTA widget with blind redirect.

* add vagovstaging to auto-deploy list (#8368)

* Fix for red Crisis Line button on homepage

* updating ebenefits module language

* updating ebenefits module copy

* changing <p> to <br>

* adding <br>

* updating ebenefits button copy

* trying to fix h1

* trying to fix bug

* bug/12987:Link under About VA in top nav goes to "This site can’t be reached": add www to link

* Fixes for #13017 Linking issues on Disability Left Nav

* Fixes top nav links for burals and education

* bug/13016: Linking Issues on Disability Hub Page: removes target blank

Removes target="_blank" on va.gov pages on disability hub

* bug/13030: Apply for benefits link on Related diseases page points to
vets.gov: Fixed url

Url now points to current site

* bug/13011: How to Use GI Bill Benefits Page: Set up order

Fixes missing order attribute and spelling error in
brand-consolidation.json file

* bug/12992: Family and Caregiver link isnt working on the home page: updated link

* 13019: Linking issues on Education Hub Page: Fix education hub

* 13020: Linking issues on Education Left Nav: Fixed education left nav

* 13043: Linking issues in Health Care left nav: Healthcare left nav

Healthcare left nav updated and accurate

* 13044:  Linking issues in Health Care top nav: Healthcare top nav fixed

* fix for build fail for unresolved links

* 13022: Linking issues in Burials Hub page: Fix burials hub page

* 12999: Huge space before period in yellow homeless box: css fix

* bug/12990: Old Check Post-9/11 GI Bill tool showing on bottom of new page: temporary fix until product figures out the flow

* 13012: Pay.gov link on health hub open in new tab

* 12982: Not enough space in displaying logged in username: Username
spacing

* 12989 - fix facilities button

* bug/13055: loading spinners broken: added missing spinner asset

* bug/13062: Top task boxes on homepage are not the same size on desktop: add min-height to fix this

* removed extra display title

* remove devpreview env (#8383)

* Port search.gov from vets.gov to preview.va.gov (#8385)

* Add isBrandConsolidationActive feature flag to SearchMenu
* Establish search attributes to component state

* feature/12735:Remove "File a Claim for Disability Compensation" From Disability Top Nav Menu: removed links associated to file a claim page (#8300)

* Configures preview for Vet360 updates (#8418)

* 12875: Update benefit hub admin. box: Enable benefit hub admin box (#8397)

* 12875: Update benefit hub admin. box: Enable benefit hub admin box

* lint

* include social accordion as metalsmith include

* IIFE

* creating page (#8286)

* creating page

* reworking copy

* minor formatting tweaks

* updating intro

* tweaks to copy/adding blue box

* adding question and tweaking intro

* tweaks

* ME review and copyedit: change-gi-bill-benefits

* Fix burial side nav (#8429)

* fixed Apply for education benefits link showing up innav (#8428)

* remove _blank on burials hub (#8424)

* clean up order of health side-nav (#8426)

* update DRC link for disability hub (#8427)

* records hub side-nav

* clean up education hub page

* cleans up link in hub page, homepage and top nav

* links all pages

* adds in records pages

* adding disability and education links to homepage (#8425)

* adding disability and education links to homepage

* changing apply for home loan coe link

* updating disability increase link

* Brand consolidation - Implement URL mappings for Life Insurance, Pensions, Housing, Careers,  (#8434)

* Process mappings for Pensions, Careers, Life Insurance, and Housing

* Fix floating link to disability

* Delete employment directory

* Fix up broken employment links

* Delete all unused life insurance directories

* Remove unused pension directory

* remove unused housing-assistance

* Remove more unused

* Replace /housing/ links

* Remove conflicted health-care/secure-messaging/index

* Va gov/social box for bc (#8444)

* 12875: Update benefit hub admin. box: Enable benefit hub admin box

* lint

* include social accordion as metalsmith include

* IIFE

* update social box to match proper styles

* account for padding in accordion content

* removed file an appeal form the MegaMenu for Pensions

*  fix megamenu spacing and scrolling

* remove border in mobile footer

* add housing left rail nav

* updated icon for housing

* change check appeal to # link

* creating va-gov/pages/claim-or-appeal-status.md (#8431)

* creating page

* Tweaking format/content

* adding bolding

* tweaks

* updating url

to match title ("and" to "or")

* tweaks

* ME review and copyedit: claim-or-appeal-status

* tweaking for SEO and replacing button with module (#8432)

* tweaking for SEO and replacing button with module

* minor tweak

* tweak

* ME review and copyedit: download-VA-letters

* changing template

* add pension left rail nav: Pensions: Create Sidebar (#8456)

* add pension left rail nav

* add pension icon to collections

* update broken link for build

* update housing on homepage links (#8458)

* 12982: Not enough space in displaying logged in username: Fix username (#8459)

spacing

* Va gov/megamenu populate links (#8461)

* update mega menu links

* upgrades design system for target blank

* Open sign-in modal with the mega-menu sign-in button. (#8462)

* Va gov/megamenu populate links (#8477)

* update mega menu links

* upgrades design system for target blank

* megamenu link fixes

* Change /facilities to /find-locations (#8474)

* Va gov/rw footer update (#8415)

* footer links

* cleans up open veterans crisis modal

* cleans up footer-links.json

* cleans up footer code

* cleans up event listener

* brought in accessiblity helpers

* fixed order, and contact us links in footer

* allow for keypress on veterans crisis selected

* update footer link orders

* update footer-links

* bottom rail links

* Convert Preview Alert to Metalsmith include (#8476)

* converted preview alert to static template and adjusted crisis line position

* converted preview alert to static template and adjusted crisis line position

* remove preview alert component from index.js

* Va gov/content wip apply for adaptive housing (#8390)

* updating title, url, and link to how to apply

* Creating page

* fixing formatting

* updating content

* updating display title

* removing extra <div>s

* updating formatting

* adding children

* tweaks

* tweaking format

* fixing formatting

* Update disability-housing-grants.md

* formatting tweaks

* adjusting title for length

* adjusting title

* fixing button adding metadata

* adding button and metadata

* trying to fix formatting

* fixing alias

* removing "collection"

* Update disability-housing-grants.md

* copy tweaks

* Update disability-housing-grants.md

* ME review and copyedit of disability-housing-grants

* ME review and copyedit of how-to-apply SAH

* Update how-to-apply.md

* updating check claim status link

* Fix padding-right on external links w/.usa-button-primary

* Fix typo in collection name

* Update collection names

* Update display_title attribute

* add spoke and corrected child collection

* replaced /facilities/ with #

* Va gov/content wip file appeal for tsgli landing (#8402)

* creating page

* adding spaces

* Update file-appeal-for-tsgli.md

* tweaks

* copy tweaks

* tweaks

* minor tweak

* minor tweak

* ME review 1: TSGLI

* Update file-appeal-for-tsgli.md

* Update file-appeal-for-tsgli.md

* open download link in new window

* updating link for file appeal

* add in proper order and spoke for page

* duplicate spokes

* Populate contact-us links

* Va gov/content wip change va direct deposit contact info landing page (#8408)

* Creating page

* fixing phone number

* adding ebenefits module

* content and formatting tweaks

* tweaks

* ME review and copyedit: change-direct-deposit

* Update change-direct-deposit-and-contact-information.md

* Update change-direct-deposit-and-contact-information.md

* Update change-direct-deposit-and-contact-information.md

* adding button link to ebenefits

* updating link to change direct deposit

* updating change direct deposit link

* updating change direct deposit link

* Va gov/content wip get military records landing page (#8454)

* creating page

* adding more content

* fixing formatting

* fixing bolding

* tweaks

* ME review and copyedit: get military records

* adding link for ebenefits button

* updating military records link

* Va gov/megamenu mobile bugfix (#8492)

* 12875: Update benefit hub admin. box: Enable benefit hub admin box

* lint

* include social accordion as metalsmith include

* IIFE

* update social box to match proper styles

* account for padding in accordion content

*  fix megamenu spacing and scrolling

* cleans up mobile nav

* Careers: Create sidebar (#8494)

* adding pages to left rail nav for career-employment

* add all pages to left rail nav

* Remove family caregiver (#8500)

* removing family & caregiver

* removing family & caregiver

* removing family & caregiver

* Va gov/life insurance hub (#8497)

* gets side nav functioning for life-insurance pages

* cleans up accordions

* update links on Life insurance hub

* Va gov/accordion fix (#8496)

* cleans up conflicting uswds.min.js imports

* makes accordions multi-expandable

* fixes accordion button styles

* Brand consolidation - Implement "reverts" for VA.gov forms URLs back to Vets.gov URLs (#8506)

* Replace privacy URLs

* Rename to welcome-kit

* Revert /find-locations/ back to /facilities/

* Delete the VA.gov forms pages

* Move the Vets.gov pages back over

* Revert form-1990

* Revert links for form-22-5490

* revert links to form-22-1990e

* Revert links to 1990N

* Revert /education/information-sharing-opt-out/

* Revert gi-bill-comparison-tool

* Cope over the health-care apply directory

* Delete the old health-care apply

* Revert health-care apply links

* Revert disability-benefits forms

* Revert links to /disability/file-for-increase-form-21-526ez/

* Revert links to /disability/add-remove-dependent/

* Move back the Burials forms

* Revert Burials links to Vets.gov links

* Revert discharge-upgrade

* revert links to discharge-upgrade-instructions

* Revert Pension 527EZ

* revert links to Pension 527ez

* Revert back to track-claims

* Revert links from claim-or-appeal-status/ to track-claims

* Fix all broken links

* Remove aliases from 530

* Remove aliases from 10007

* Remove aliases from discharge-upgrade-instructions

* Add back order, hideFromSidebar

* Add back the title

* Add back all frontmatter, minus the aliases

* Add back all front matte,  minus the aliases

* Add back all front matter, minus the aliases

* Add back old frontmatter

* Add back all frontmatter, minus the aliases

* Remove double display_title

* Add back all front matter, minus the aliases

* Revert root URL for GI Bill

* Replace health-care/apply root url

* Revert disability link in mega menu

* Revert burials root url

* Revert discharge upgrade instructions

* Revert pension link

* add slash to introduction on about-gi-bill

* added slash to introduction page

* Fix issue with url due to find and replace

* updating vic and home loan coe links/descriptions (#8472)

* updating vic and home loan coe links/descriptions

* tweaks

* updates

* Update top-nav links according to spreadsheet (#8507)

* Remove external link icon in sidebar and homepage-hub (#8505)

* Remove pageMetaData from eslint

* Remove the left-rail-nav

* Remove unused .ai file

* Fix manifest.json imports

* Checkout master content

* Reverts back to Vets.gov health-care links in the src/ directory

* Updates for compatibility w/Vets.gov

* Revert files back to Vets.gov versions

* Wrap widget-startups in feature flag

* Revert back to Vets.gov version

* Remove broken tests

* Revert back to Vets.gov version

* Remove unused

* Revert back to Vets.gov versions

* Remove slider.scss

* Create separate brand consolidation base style

* Add the announcement-root

* Revert back to master version

* Change fa-handshake to fa-handshake-o for fontawesome4 compat

* Remove fontawesome 5

* Add VA.gov environments

* Revert "Add VA.gov environments"

This reverts commit 26e89d45d6f1fc80abfdd9341608982b82576c97.

Commit:
Merge origin/brand-consolidation

Commit:
Merge remote-tracking branch 'origin/master' into va-gov/feature/13411

RobAWilkinson reviewed (approved):
LGTM

accbjt committed:
add space for building on Jenkins

accbjt committed:
resolved merge conflicts
